docs/advanced-scenarios.md

```diff
@@ -0,0 +1,150 @@
+<!-- meta
+{
+  "node": {
+    "label": "Expert scenarios",
+    "category": {
+      "label": "Advanced"
+    },
+    "metaData": {
+      "categoryPosition": 4,
+      "position": 1
+    }
+  }
+}
+meta -->
+
+# Expert scenarios
+
+This is a collection of advanced use cases and example implementations. If you are new to Luigi, take a look at our [Getting Started](getting-started.md) section first.
+
+<!-- accordion:start -->
+
+### Use a SPA router and keep Luigi Core URL in sync
+
+#### Overview
+
+This example shows you how to keep an existing routing strategy and use an existing micro frontend as drop-in without the need to refactor everything to `LuigiClient.linkManager()`. To update the Luigi Core URL when routing internally with the micro frontend router, without updating the URL on the Luigi Client side, use the `linkManager()` [withoutSync](luigi-client-api.md#withoutsync) method.
+
+<!-- add-attribute:class:warning -->
+> **NOTE**: This is a very simple example for navigation. Special cases like modals, split views and other functionalities still require the use of [Luigi Client](luigi-client-api.md).
```

Comment:
```suggestion
> **NOTE**: This is a very simple example. For cases like modals, split views, and other functionalities you still have to use [Luigi Client](luigi-client-api.md).
```
Maybe we can have a third specific case instead of `other functionalities`? If not, I would just put ` For cases like modals or split views, you still...`